cognite/client/_api/data_modeling/instances.py

```diff
@@ -349,8 +350,14 @@ def _create_other_params(
         return other_params
 
     @classmethod
-    def _dump_instance_source(cls, source: ViewId) -> dict:
-        return {"source": source.dump(camel_case=True)}
+    def _dump_instance_source(cls, source: ViewIdentifier) -> dict:
+        instance_source: ViewId
+        if isinstance(source, ViewId):
+            instance_source = source
+        if isinstance(source, tuple):
+            instance_source = ViewId(*source)
+
+        return {"source": instance_source.dump(camel_case=True)}
```

Comment:
ViewId:load already does this
```suggestion
        return {"source": ViewId.load(source).dump(camel_case=True)}
```

Reply:
Fixed in new commit. Also had to make changes to ids.py for mypy